cluster/ci/config/prow/config.yaml

```diff
@@ -181,3 +181,308 @@ tide:
+        ci-vm-operator:
+          protect: true
+          enforce_admins: true
+        openshift-azure:
+          protect: true
+          enforce_admins: true
+        elasticsearch-operator:
+          protect: true
+          enforce_admins: true
+        coredns:
+          protect: true
+          enforce_admins: true
+        ci-ns-ttl-controller:
+          protect: true
+          enforce_admins: true
+        cluster-logging-operator:
+          protect: true
+          enforce_admins: true
+        online-console-extensions:
+          protect: true
+          enforce_admins: true
+        cluster-version-operator:
+          protect: true
+          enforce_admins: true
+        machine-config-operator:
+          protect: true
+          enforce_admins: true
+        builder:
+          protect: true
+          enforce_admins: true
+        cluster-network-operator:
+          protect: true
+          enforce_admins: true
+        cluster-image-registry-operator:
+          protect: true
+          enforce_admins: true
+        online-registration:
+          protect: true
+          enforce_admins: true
+        kubernetes-metrics-server:
+          protect: true
+          enforce_admins: true
+        release:
+          protect: true
+          enforce_admins: true
+        prometheus-operator:
+          protect: true
+          enforce_admins: true
+        cluster-openshift-apiserver-operator:
+          protect: true
+          enforce_admins: true
+        telemeter:
+          protect: true
+          enforce_admins: true
+        cluster-kube-apiserver-operator:
+          protect: true
+          enforce_admins: true
+        jenkins-plugin:
+          protect: true
+          enforce_admins: true
+          required_status_checks:
+            strict: true
+            contexts:
+            - continuous-integration/travis-ci/pr
+        jenkins-sync-plugin:
+          protect: true
+          enforce_admins: true
+          required_status_checks:
+            strict: true
+            contexts:
+            - continuous-integration/travis-ci/pr
+        library:
+          protect: true
+          enforce_admins: true
+          required_status_checks:
+            strict: true
+            contexts:
+            - continuous-integration/travis-ci/pr
+        autoheal:
+          protect: true
+          enforce_admins: true
+          required_status_checks:
+            strict: true
+            contexts:
+            - pull-autoheal-check
+        cluster-monitoring-operator:
+          protect: true
+          enforce_admins: true
+          required_status_checks:
+            strict: true
+            contexts:
+            - continuous-integration/travis-ci/pr
+        cluster-openshift-controller-manager-operator:
+          protect: true
+          enforce_admins: true
+          required_status_checks:
+            strict: true
+            contexts:
+            - continuous-integration/travis-ci/pr
```

Comment:
@stevekuznetsov why is this forcing travis for people? if you remove .travis.yaml this will still require it, right? that's strange since they've never configured it in this repo.